### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][Keyboard][MAS2.4.7][IntelliJ][Wipro]: Keyboard focus is not visible on plugin related information in tab plugins dialog.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1455]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Navigate and select Settings Menu item under File menu.
3. In Settings dialog  box, navigate and select Plugins.
4. Plugins dialog will open.
5. In plugins dialog Select any particular plugin(Ex: GitHub Integration).
6. After selecting the plugin navigate to plugin information using "Ctrl+tab" and observe keyboard focus.

**Expected Result:**
Keyboard focus should be  visible on  plugin related information tab in plugins dialog.

**Actual Result:**
Keyboard focus is not visible on plugin related information tab in plugins dialog.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![focus not visible on plugin information tab](https://user-images.githubusercontent.com/22956130/35801645-f6f646de-0a21-11e8-8584-4b23be0ccb32.png)

**Comment by Eskibear** [comment]
The dialog is IntelliJ IDEA's own setting page, it's out of the toolkit's scope.

**Comment by Eskibear** [comment]
tracked in https://youtrack.jetbrains.com/issue/IDEA-191938

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. This is the feature of IntelliJ IDEA itself. It's out of the plugin scope.
* **Quality of experience**: not impacted. This cannot be controlled by the plugin since it is the IntelliJ IDEA's feature. But if the user do want to see the introduction of any IntelliJ plugins, he can view the online docs for IntelliJ. 
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749885
